app/modules/users/resources.py

```diff
@@ -409,14 +406,25 @@ def post(self):
             # Log for development purposes but do not show anything to the API user
             log.warning('User with email address {repr(email)} not found')
             return
-        code = Code.get(user, CodeTypes.recover, replace=True)
+        code = user.get_account_recovery_code()
         msg = Email(recipients=[user])
         url_prefix = msg.template_kwargs['site_url_prefix']
         reset_link = urljoin(url_prefix, f'/auth/code/{code.accept_code}')
         msg.template('misc/password_reset', reset_link=reset_link)
         msg.send_message()
 
 
+@api.route('/<uuid:user_guid>/verify_account_email')
```

Comment:
am i reading it right this has no permissions?   in a way that makes sense -- but might also be subject to potential abuse later.   can a user login _without_ this verification?

Reply:
I'll remove `<uuid:user_guid>/` and change this API to logged in user only